### ISSUE TITLE
Bug: Gauge chart always adds at least three table cells

### ISSUE THREAD

**Issue by XtolotosX** [google/google-visualization-issues#2244]
Just did some testing with the gauge chart and discovered that it always adds three table-cells for it's content even if there is only data for two and only two meters are shown.

Example here: https://jsfiddle.net/s8w5jgvn/

**Comment by dlaliberte** [comment]
You are correct.  This is peculiar, and only seems to happen when you have two rows of data.
